ast/compile.go

```diff
@@ -1277,6 +1284,39 @@ func (c *Compiler) getExports() *util.HashMap {
 	return rules
 }
 
+func (c *Compiler) checkDuplicateImports() {
+	if !c.strict {
+		return
+	}
+
+	for _, name := range c.sorted {
+		mod := c.Modules[name]
+		processedImports := map[Var]*Import{}
+
+		for _, imp := range mod.Imports {
+			var name Var
+
+			if len(imp.Alias) > 0 {
+				name = imp.Alias
+			} else {
+				path := imp.Path.Value.(Ref)
+				if len(path) == 1 {
+					name = path[0].Value.(Var)
+				} else {
+					v := path[len(path)-1].Value.(String)
+					name = Var(v)
+				}
+			}
+
+			if processed, conflict := processedImports[name]; conflict {
+				c.err(NewError(CompileErr, imp.Location, fmt.Sprintf("import must not shadow %v", processed)))
```

Comment:
https://github.com/johanfylling/opa/blob/b90a7237e7c89c867b92a5c823aed20c7184d473/ast/errors.go#L117 I think that `NewError` lets us do this here:
```suggestion
				c.err(NewError(CompileErr, imp.Location, "import must not shadow %v", processed))
```